mobile/src/components/chat-conversation/components/message-form/components/send-button/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  button: {
+    width: 40,
+    height: 40,
+    alignSelf: 'center',
+    alignItems: 'center',
+    justifyContent: 'center',
+    marginRight: 5,
+  },
+  disabledButton: {
+    opacity: 0.5,
+  },
+  iconShadow: {
```

Comment:
Why do we need this and why is it absolute?

Reply:
It's absolute to fully cover the icon, because as I understood, the keyboard is hiding when the click is over the icon

Reply:
it is also the reason why I used so weird background color, because 'transparent' or its absence will trigger hit on the icon

Reply:
This is how I understood the reason of this problem)

Reply:
Can't we use same approach as for `InputSecure`

Reply:
I've tried and it had the same result - the keyboard is hiding when I hit the icon. I've already tried lots of variants) And only this one in PR was working well. IDK why